src/com/lftechnology/java/training/alina/basic/multiplesum/SumOfMultiples.java

```diff
@@ -4,50 +4,46 @@
 import java.util.logging.Logger;
 
 /**
- * If we list all the natural numbers below 10 that are multiples of 3 or 5, we
- * get 3, 5, 6 and 9. The sum of these multiples is 23. Find the sum of all the
- * multiples of 3 or 5 below 1000.
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23. Find
+ * the sum of all the multiples of 3 or 5 below 1000.
  * 
  * @author Alina Shakya<alinashakya@lftechnology.com>
  */
 public class SumOfMultiples {
-	private static final Logger LOGGER = Logger.getLogger(SumOfMultiples.class
-			.getName());
+    private static final Logger LOGGER = Logger.getLogger(SumOfMultiples.class.getName());
 
-	public static void main(String[] args) {
-		int maxNum = 1000;
-		int num1 = 3;
-		int num2 = 5;
-		try {
-			getTotalSum(maxNum, num1, num2);
-			int totalSumOfMultiples = getTotalSum(maxNum, num1, num2);
-			LOGGER.log(Level.INFO, "Sum of multiples of {0} and {1} is : {2}",
-					new Object[] { num1, num2, totalSumOfMultiples });
-		} catch (ArithmeticException ex) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}",
-					new Object[] { ex.getMessage() });
-		}
-	}
+    public static void main(String[] args) {
+        int maxNum = 1000;
+        int num1 = 3;
+        int num2 = 5;
+        try {
+            getTotalSum(maxNum, num1, num2);
+            int totalSumOfMultiples = getTotalSum(maxNum, num1, num2);
+            LOGGER.log(Level.INFO, "Sum of multiples of {0} and {1} is : {2}", new Object[] { num1, num2, totalSumOfMultiples });
+        } catch (ArithmeticException ex) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)